### PR TITLE
[chore](vulnerability) fix fe high risk vulnerability scanned by bug scanner

### PR DESCRIPTION
**Pull request by yangzhg**
## Problem summary

fix fe high risk vulnerability scanned by bug scanner

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [x] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 36.83 seconds
 load time: 655 seconds
 storage size: 17123621264 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230105130726_clickbench_pr_74383.html

File: fe/fe-core/src/main/java/org/apache/doris/common/util/DynamicPartitionUtil.java [hunk 1]
```diff
@@ -740,8 +740,7 @@ public static String getHistoryPartitionRangeString(DynamicPartitionProperty dyn
             date = simpleDateFormat.parse(time);
         } catch (ParseException e) {
             LOG.warn("Parse dynamic partition periods error. Error={}", e.getMessage());
-            return getFormattedTimeWithoutMinuteSecond(
-                    ZonedDateTime.parse(timestamp.toString(), dateTimeFormatter), format);
+            throw new AnalysisException("Parse dynamic partition periods error. Error=" + e.getMessage());
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/common/util/DynamicPartitionUtil.java:743]: Why changing this logic?

Comment (yangzhg) [reply fe/fe-core/src/main/java/org/apache/doris/common/util/DynamicPartitionUtil.java:743]: because of timestamp is null at this time

**Review by morningman — DISMISSED** [review]
LGTM

**Review by xy720 — DISMISSED** [review]
LGTM

**Review by xy720 — DISMISSED** [review]
LGTM

**Review by xy720 — APPROVED** [review]
LGTM

**Pull request merged**